bin/pftt2-release.bat

```diff
@@ -0,0 +1,34 @@
+@ECHO OFF
+
+REM Check if build directory exists
+cd ../
+if exist build (
+	REM Create pftt-release directory in main folder
+	mkdir pftt-release
+
+	REM Create pftt2.jar and place in lib folder
+	cd build
+	jar cf pftt2.jar ./com/ ./org/columba ./org/kxml2 ./org/incava
+	MOVE ./pftt2.jar ../lib
+	cd ..
+
+	REM Copy contents of bin, conf and lib to respective folders
+	cd /d pftt-release
```

Comment:
It seems we don't really need the /d option here, but it is not a big deal though...

Reply:
Easy minor change. Thanks